js/bitfinex2.js

```diff
@@ -458,7 +512,21 @@ module.exports = class bitfinex2 extends bitfinex {
                     },
                 },
                 'info': market,
-            });
+            };
+            if (swap) {
+                const settlementCurrencies = this.options['swap']['fetchMarkets']['settlementCurrencies'];
+                for (let i = 0; i < settlementCurrencies.length; i++) {
```

Comment:
None of the markets settled in BTC show up, just the USDT ones

Reply:
I think the problem might be that the ids for the USDT and BTC markets are the same, so when the ids have `indexBy`  run on them, then they're overwritten

Reply:
@kroitor We should probably still have both though